src/layout/Navbar/NavButton.svelte

```diff
@@ -26,11 +27,13 @@
 	}
 </script>
 
-<a data-sveltekit-preload-data on:mouseenter={prefetch} {href}>
+<a data-sveltekit-preload-data on:mouseenter={prefetch} {href} aria-label={label}>
 	<!-- Check if href is equal to the first path -->
-	<li class:selected={href === '/' + $RouterEvents.target_url.pathname.split('/')[1]}>
-		<span><slot /></span>
-	</li>
+	<ul>
+		<li class:selected={href === '/' + $RouterEvents.target_url.pathname.split('/')[1]}>
+			<span><slot /></span>
+		</li>
+	</ul>	
```

Comment:
Why create an unordered list for each listitem? maybe you can move <ul> to NavHost

Reply:
Hi! 
I've tried wrapping the buttons in NavHost like this : 
<img width="694" alt="Captura de Tela 2023-11-22 às 15 13 40" src="https://github.com/ReVanced/revanced-website/assets/64154960/0ac08690-296e-4305-81f3-698d19d38eb1">

but It doesn't seem to fix 
![Captura de Tela 2023-11-22 às 15 20 09](https://github.com/ReVanced/revanced-website/assets/64154960/05eeddc5-2cc2-4bbd-b584-1593b7b132a6)

Reply:
It may be because the a href comes before the li, can you try reordering them and checking again? if it fixes the issue, make sure to adjust the css to make sure it acts the same

Reply:
Yes, I'll give it a try.

Reply:
Reordering the elements solved the issue. I've committed the changes. I've also refactored the CSS to account for these changes and removed some redundant rules.